lib/components/helpers/pretty-text-input.js

```diff
@@ -40,7 +40,16 @@ export default React.createClass({
   },
 
   componentDidUpdate: function(prevProps, prevState) {
-    if (prevState.codeMirrorMode !== this.state.codeMirrorMode) {
+    const wasPreviewing = prevProps.field.meta.isPreviewing;
```

Comment:
The `meta` property is only available when the meta plugin is enabled. So you can't depend on that being here. And if you look around, you'll see that none of the formatic's components know about the meta plugin at all.

Beyond that, I think "preview" is a zapier-ism, and sniffing a change of that value to "kick" codemirror is kind of smelly.

I think the real problem is that you're not seeing codemirror update when you change the value, and we should probably just fix that. If there are perf problems with that, we'll just have to involve a debounce or something to tame it.